### PR TITLE
Add support for TLS verification skipping in Podman logins.

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date

### CODE REVIEW

**Comment by cjeanner** [comment]
The zuul crash might be related to #292 ? I've seen some discussions about that.

File: ci_framework/roles/openshift_setup/tasks/main.yml [hunk 1]
```diff
@@ -82,3 +70,36 @@
...
+
+- name: Podman login into OpenShift registry
+  when:
+    - cifmw_openshift_token is defined
+    - cifmw_openshift_setup_registry_default_route.resources | length > 0
+  block:
+
+  - name: Wait for the image registry to be ready
+    kubernetes.core.k8s_info:
+      kind: Deploymeny
```
Comment (Sandeepyadav93) [inline ci_framework/roles/openshift_setup/tasks/main.yml:82]: typo: Deployment

Comment (pablintino) [reply ci_framework/roles/openshift_setup/tasks/main.yml:82]: how is this working with that typo?

File: scenarios/centos-9/edpm_ci.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ cifmw_edpm_deploy_manifests_dir: "{{ cifmw_installyamls_repos }}/out"
 deploy_edpm: true
 
 # openshift_setup role vars
-cifmw_openshift_setup_skip_internal_registry_login: true
```
Comment (Sandeepyadav93) [inline scenarios/centos-9/edpm_ci.yml:28]: Imho.. If we don't pull anything from the internal registry, imho.. we should skip the login.

**Review by cjeanner — APPROVED** [review]

**Pull request merged**